modules/ServerAPI/src/server/routes/InventoryServiceRoutes.coffee

```diff
@@ -1366,22 +1369,27 @@ exports.moveToLocationInternal = (input, callCustom, callback) ->
 				if !error
 					console.log 'container down here...did it pull through??'
```

Comment:
Can you remove this line? console.log 'container down here...did it pull through??'